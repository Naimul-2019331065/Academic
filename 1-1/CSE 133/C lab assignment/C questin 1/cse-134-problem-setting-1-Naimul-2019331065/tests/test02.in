3
7 7
4 4
15 9
5 3
10000 9999
9999 10000