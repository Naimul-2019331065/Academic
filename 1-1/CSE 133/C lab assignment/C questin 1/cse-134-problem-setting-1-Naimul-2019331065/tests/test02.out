7
9
5000